src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -344,6 +344,8 @@
   "text.autoconfig.skyblocker.option.locations.dwarvenMines.solvePuzzler": "Solve Puzzler Puzzle",
   "text.autoconfig.skyblocker.option.locations.dwarvenMines.metalDetectorHelper": "Metal Detector Helper",
   "text.autoconfig.skyblocker.option.locations.dwarvenMines.metalDetectorHelper.@Tooltip": "Helper for the metal detector puzzle in the Mines of Divan.",
+  "text.autoconfig.skyblocker.option.locations.dwarvenMines.enableGlaciteColdOverlay": "Glacite Cold Overlay",
+  "text.autoconfig.skyblocker.option.locations.dwarvenMines.enableGlaciteColdOverlay.@Tooltip": "Shows a Cold Effect in the Glacite mineshafts area of the Dwarven Mines.",
```

Comment:
Maybe instead say something along the lines of "Shows a frosty overlay in the Glacite mines that gets stronger as you get colder." which is more descriptive of how it works.